modules/installation-vsphere-installer-infra-requirements.adoc

```diff
@@ -371,7 +371,7 @@ Additionally, the user requires some `ReadOnly` permissions, and some of the rol
 
 |vSphere vCenter Resource Pool
 |Existing resource pool
```

Comment:
Hi @sgaoshang . Does the update apply to the vSphere vCenter Resource Pool object as well? I've been informed that this might not be the case. Can you run a test?